src/Uno.Toolkit.UI/Controls/ExtendedSplashScreen/ExtendedSplashScreen.Windows.crossruntime.cs

```diff
@@ -1,4 +1,4 @@
-﻿#if WINDOWS || WINDOWS_UWP || NET472_OR_GREATER || NETSTANDARD2_0
```

Comment:
CI failure:

> D:\a\1\s\src\Uno.Toolkit.UI\Controls\ExtendedSplashScreen\ExtendedSplashScreen.Windows.crossruntime.cs(40,15): error CS0102: The type 'ExtendedSplashScreen' already contains a definition for 'SplashIsEnabled' [D:\a\1\s\src\Uno.Toolkit.UI\Uno.Toolkit.UI.csproj::TargetFramework=net7.0-ios]

As it complains about this file, it means the `#if` is true on iOS. However, WINDOWS, WINDOWS_UWP, UNO_REFERENCE_API all should be false on iOS 😨

Reply:
Ok you're hitting a bug that was fixed in https://github.com/unoplatform/uno/pull/14282.

To avoid upgrading Uno, I think you can change `UNO_REFERENCE_API` to `__SKIA__ || __WASM__`

Reply:
Actually, no, `__SKIA__ || __WASM__` isn't going to work either

Reply:
Maybe add `<DefineConstants Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == ''">$(DefineConstants);__SKIA_OR_WASM__</DefineConstants>`

Then use `__SKIA_OR_WASM__` instead of `UNO_REFERENCE_API`. It would also be good to add a comment on that line saying that `UNO_REFERENCE_API` can be used once the project is upgraded to Uno 5.1

Reply:
weird, its compiling fine locally for ios

Reply:
> Maybe add `<DefineConstants Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == ''">$(DefineConstants);__SKIA_OR_WASM__</DefineConstants>`
> 
> Then use `__SKIA_OR_WASM__` instead of `UNO_REFERENCE_API`. It would also be good to add a comment on that line saying that `UNO_REFERENCE_API` can be used once the project is upgraded to Uno 5.1

@Youssef1313 Is the condition for that `DefineConstants` supposed to be `"$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == ''"` or should it be `"$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'net7.0'"`?

Reply:
Oh no nevermind I see what you're doing